libs/blocks/media/media.css

```diff
@@ -103,13 +103,97 @@ div[class*="-up"] .media .foreground > .media-row {
   padding: 0 0 var(--spacing-m);
 }
 
+.media .default-list {
+  padding-left: 20px;
+  margin-top: var(--spacing-xs);
+  margin-bottom: var(--spacing-xs);
+}
+
+.media li:before {
+  content: "";
+  display: inline-block;
+  margin-left: 3px;
+}
+
+.media.checklist li {
+  background: url("/img/checklist.svg") no-repeat 5px 5px transparent;
+  list-style-type: none;
+  margin-left: -25px;
+  padding: 0 0 1px 29px;
+  vertical-align: middle;
+}
+
+.media .subcopy {
+  font-size: var(--type-heading-xs-size);
+  line-height: var(--type-body-s-lh);
+  margin-top: var(--spacing-xs);
+  color: #2C2C2C;
+}
+
+.media .action-area {
```

Comment:
Duplicate selector. Can you move this to line 19.

Reply:
Resolved https://github.com/adobecom/milo/pull/971/commits/cc6a21365619e5afcd3e7dc6ac7814b8b4f93072 @ryanmparrish